### PR TITLE
Enable Autopilot

### PR DESCRIPTION
**Pull request by soerenmartius**
Depends on the initial import in #1

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -321,12 +249,12 @@ variable "master_ipv4_cidr_block" {
 
 variable "release_channel" {
   type        = string
-  description = "(Optional) The release channel of this cluster. Accepted values are `UNSPECIFIED`, `RAPID`, `REGULAR` and `STABLE`. Defaults to `UNSPECIFIED`."
-  default     = "UNSPECIFIED"
+  description = "(Optional) The release channel of this cluster. Accepted values are `UNSPECIFIED`, `RAPID`, `REGULAR` and `STABLE`."
+  default     = "STABLE"
```
Comment (mariux) [inline variables.tf]: why remove unspecified? 

-> you forgot to remove it also in description.

**Review by mariux — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (soerenmartius) [reply variables.tf]: because unspecified is not supported in autopilot

Comment (soerenmartius) [reply variables.tf]: will change it in the description

Comment (soerenmartius) [reply variables.tf]: fixed in bcb11652

File: main.tf [hunk 2]
```diff
@@ -9,51 +9,31 @@ resource "google_container_cluster" "cluster" {
...
-  cluster_ipv4_cidr         = var.cluster_ipv4_cidr
-  default_max_pods_per_node = var.default_max_pods_per_node
-
-  dynamic "authenticator_groups_config" {
-    for_each = var.rbac_security_identity_group != null ? [1] : []
-
-    content {
-      security_group = var.rbac_security_identity_group
-    }
-  }
```
Comment (mariux) [inline main.tf:47]: i assume we still want to use google groups for rbac? why remove it? this is still an optional feature in autopilot

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,8 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add support for `null_resource` Terraform resource
+- Add support for `google_container_cluster` with autopilot enabled
 
 [unreleased]: https://github.com/mineiros-io/terraform-google-gke-autopilot-cluster/compare/v0.0.1...HEAD
-<!-- [0.0.2]: https://github.com/mineiros-io/terraform-google-gke-autopilot-cluster/compare/v0.0.1...v0.0.2 -->
```
Comment (mariux) [inline CHANGELOG.md:17]: please do not remove this.. as this needs to be commented out when realesing 0.0.2... else people copy the tag as happened before..

**Review by mariux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (soerenmartius) [reply CHANGELOG.md:17]: resolved in d8cae50d

**Review by mariux — APPROVED** [review]
lgtm

File: main.tf (hunk 2 above)
Comment (soerenmartius) [reply main.tf:47]: As discussed, I will open an issue in https://github.com/hashicorp/terraform-provider-google

**Pull request merged**